jsonschema/docs/docs_test.go

```diff
@@ -0,0 +1,33 @@
+//go:build !windows
+// +build !windows
```

Comment:
Not a major issue, but can we handle `\r\n` so that developers using Windows can still run the tests? We do this here for example: https://github.com/cloudquery/cloudquery/blob/main/cli/cmd/publish.go#L363-L368